java/registry/src/main/java/dev/sunbirdrc/registry/controller/RegistryEntityController.java

```diff
@@ -450,6 +450,7 @@ private ObjectNode copyWhiteListedFields(ArrayList<String> fields, JsonNode data
             node = objectMapper.readTree(node.get(OSSystemFields._osSignedData.name()).asText());
             return new ResponseEntity<>(certificateService.getCertificate(node,
                     entityName,
+                    entityId,
```

Comment:
the node object will already contain entityId or Osid,
do we still need to pass it externally?